arm-gateway/2016-06-01/swagger/gateway.json

```diff
@@ -0,0 +1,442 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "GatewayManagementClient",
+    "description": "REST API for On-Premises Data Gateways.",
+    "version": "2016-06-01"
```

Comment:
is this the correct API version? (just asking since it's been almost a year from that date)
Is this an existing service just on-boarding to swagger?